packages/css/src/components/header/header.scss

```diff
@@ -0,0 +1,62 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+@import "../../common/breakpoint";
+
+.amsterdam-header {
+  align-items: center;
+  background-color: var(--amsterdam-header-background-color);
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  padding-block: 0.88rem;
+  row-gap: 1.5rem;
+
+  @media screen and (max-width: $amsterdam-breakpoint-medium) {
+    column-gap: 0;
+
+    *[class*="page-menu__item"]:not([class$="--mobile"]) {
+      display: none;
+      visibility: hidden;
+    }
+  }
+
+  @media screen and (min-width: $amsterdam-breakpoint-wide) {
+    column-gap: var(--amsterdam-header-column-gap);
+    flex-wrap: nowrap;
+  }
+}
+
+.amsterdam-header__column--logo {
+  flex: 0 0 auto;
+}
+
+.amsterdam-header__column--title {
+  flex: 1 1 100%;
+
+  @media screen and (min-width: $amsterdam-breakpoint-wide) {
+    flex: 1 1 auto;
+    min-width: 0;
+    order: 2;
+
+    .amsterdam-heading {
+      display: block;
+      overflow: hidden;
+      text-overflow: ellipsis;
+      white-space: nowrap;
+      width: 100%;
+    }
+  }
+}
+
+.amsterdam-header__column--menu {
+  flex: 2 1 calc(100% - 160px);
```

Comment:
What’s the 160px for?

Reply:
Maximum width of the logo, the column would try to occupy that space instead of `auto` where it would divide the available space with the title.

Reply:
Mistake, it was the width the logo would like to have but this is the menu..